core.js

```diff
@@ -2761,7 +2761,190 @@ HTMLAudioElement.HAVE_METADATA = HTMLMediaElement.HAVE_METADATA;
 HTMLAudioElement.HAVE_CURRENT_DATA = HTMLMediaElement.HAVE_CURRENT_DATA;
 HTMLAudioElement.HAVE_FUTURE_DATA = HTMLMediaElement.HAVE_FUTURE_DATA;
 HTMLAudioElement.HAVE_ENOUGH_DATA = HTMLMediaElement.HAVE_ENOUGH_DATA;
+class Video {
```

Comment:
Is this best called "Video" or more like "MediaDevices"?

Reply:
One benefit of calling it Video is that there are many points in the code where you want to iterate over only the video devices, and not e.g. audio inputs. MediaDevices would blur that line.

Reply:
Yeah I can see that. But there is `Image` and `Audio`, which are aliases for `HTMLImageElement` and `HTMLAudioElement`. Presumably `Video` would be an alias for `HTMLVideoElement`?

In which case the relationship to media devices probably wouldn't make sense.

Reply:
Honestly the only thing that we need from `Video` is `getMediaDevices()`, which can go pretty much anywhere. The only reason it was named this is because it's called `Video` in the nativeBindings. Feel free to suggest whatever changes you want.

Reply:
Rename to `MediaStream`?